docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "116af785-a9b7-4eaf-8a86-b1eeb235da18",
+   "metadata": {},
+   "source": [
+    "# Fractional gates\n",
+    "\n",
+    "This page introduces two newly supported gate types on the IBM Quantum fleet of QPUs. These *fractional* gates are supported on:\n",
+    "- [Eagle](/guides/processor-types#eagle) QPUs as $R_{ZZ}(\\theta)$ and $R_X(\\theta)$\n",
+    "- [Heron](/guides/processor-types#heron) QPUs as $R_{ZX}(\\theta)$\n",
+    "\n",
+    "This page discusses the usecases which enable your quantum workflows to become more efficient using these fractional gates, as well as how to use them on IBM Quantum QPUs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee241e99-d095-4cb5-9506-da3d92c32e1c",
+   "metadata": {},
+   "source": [
+    "## How to use fractional gates\n",
+    "\n",
+    "Internally, these fractional gates work by dynamically modifying the microwave pulses that are sent to individual qubits based on the angle, $\\theta$, that a circuit instruction specifies. This has the advantage of shorter gate durations -- and thus lower dephasing error, especially for small angles -- which reduces the overall circuit duration, in addition to the cost savings of executing fewer gates.\n",
```

Comment:
This is bit misleading because we don't shorten pulse duration with the modification. We can reduce gate count by directly implementing RZZ and RX, and thus we can reduce the circuit duration. Gate duration will be the same.